release/rllib_tests/learning_tests/hard_learning_tests.yaml

```diff
@@ -327,13 +328,13 @@ dqn-breakoutnoframeskip-v4:
         num_gpus: 0.5
         timesteps_per_iteration: 10000
 
-es-humanoid-v2:
-    env: Humanoid-v2
+es-humanoidbulletenv-v0:
```

Comment:
Wrong env specified (must be PyBullet, not MuJoCo).

Reply:
sounds good